### DISCUSSION TITLE
Kickstart network definition in pre section doesnt work

### DISCUSSION THREAD

**Discussion by ozgyilmaz** [rhinstaller/anaconda#4142] [General]
Hello,

I've tested defining network interface (set ip, hostname, subnet, gw, nameserver) in %pre section of a kickstart file and tried installing RHEL. It didnt  work and on most tested versions dracut hung.

When i cancelled the %pre section and define the network in main body of kickstart, everything worked well.

I've observed this problem with official boot images of RHEL7.1, RHEL7.2, RHEL7.8, RHEL7.9 and RHEL8.4

I've also created a bug record with id 2089456.

Thank you.

**Comment by rvykydal** [comment]
Please see the comment in the BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2089456#c5
